### ISSUE TITLE
Missing coverage information in top-level addons in 0.3.3.

### ISSUE THREAD

**Issue by nfc036** [kategengler/ember-cli-code-coverage#53]
Starting from Version 0.3.3 the parameter `name` in `_filterOutAddonFiles` is a function and not the actual name. This issue seems fixed when changing [line 255](https://github.com/kategengler/ember-cli-code-coverage/blob/0cdcccecc2d1130c551c18ea6fa1f955c07637cb/index.js#L255) (function _parentName) from 
```
return this.parent.name;
```
to
```
return this.parent.pkg.name;
```

**Comment by nfc036** [comment]
... or
```
return this.parent.name();
```
as in kategengler/ember-cli-code-coverage#52

**Comment by rwjblue** [comment]
So the underlying issue is tha `this.parent` is EITHER the project instance (which has a name function) OR an addon instance (which has name as a string property). 

The current logic seems like it will work properly for nested addons (because parent.name will be a string), but not for top level addons.

**Comment by rwjblue** [comment]
Closed by #52
